angular/_darkTheme.scss

```diff
@@ -60,11 +60,6 @@
         }
     }
 
-    /*list item*/
-    .mat-list-item-content {
```

Comment:
I guess I'm still a little confused why we need to take this rule out. Isn't this just setting a default color for the ILI text items? Shouldn't the showcase be able to override this to do the status colors?

Reply:
The showcase is able to override these styles, but they won't be able to do it conveniently but providing a [color] on the the top-level InfoListItem.   They would have 2 options, either use css with a higher specificity to override these colors, or apply the [color] directly to the icon & text they want to style.

This style isn't needed since the default color is already foreground-text.

Reply:
```
 <pxb-info-list-item dense="true" [style.color]="colors.red[500]" class="card-body-item">
      <div pxb-title>1 Alarm</div>
      <mat-icon pxb-icon>notifications</mat-icon>
</pxb-info-list-item>
```

Before this theme change, the top-level [style.color] is being overwritten by the now-removed style.

Reply:
Okay, I gotcha. Would it make more sense to apply the default color to the InfoListItem root and then set the content color to inherit (https://github.com/pxblue/themes/blob/dev/angular/pxb-component-theme.scss#L57) - hero ChannelValue and EmptyState do it that way.

```
@mixin pxb-info-list-item-theme($theme) {
    $foreground: map-get($theme, foreground);
    .pxb-info-list-item {
        color: map-get($foreground, text);
        .mat-list-item-content {
            color: inherit;
        }
    }
}
```
That way we definitely control the default color and aren't at the mercy of inheriting from somewhere else up the tree, but a user can still override it easily with `[style.color]` on the component.

Reply:
Okay, I've updated the pxblue-component-theme file to set `inherit` on the .mat-list-item-content class.  I tested this out in the showcase app, works just as expected.

Reply:
Looks like we are still missing the default color rule `color: map-get($foreground, text);` on the `.pxb-info-list-item`

Reply:
Oops, I forgot this.  Added it back, re-tested showcase.